api/db/changelog/db.changelog-107-workspace-free-tier-usage-user-index.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="thibault" id="changelog-107-workspace-free-tier-usage-user-index">
+    <createIndex tableName="workspace_free_tier_usage" indexName="workspace_free_tier_usage_user_idx" unique="false">
+      <column name="user_id"/>
```

Comment:
Added a query to this table with "WHERE user = foo"

Reply:
👍 indexes ! we don't use them enough